src/components/frame/Extensions/pages/ExtensionDetailsPage/PermissionsNeededBanner.tsx

```diff
@@ -43,6 +43,7 @@ const PermissionsNeededBanner = ({ extensionData }: Props) => {
     extensionData.extensionId ?? '',
   );
 
+  const [shouldDisplay, setShouldDisplay] = useState(false);
```

Comment:
I think this can just be a variable like
`const shouldDisplay = extensionData.extensionId === Extension.MultisigPermissions` and then we don't need to set state